.github/workflows/gettext.yaml

```
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: 2023 Carmen Bianca BAKKER <carmenbianca@fsfe.org>
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+name: Update .pot file
+
+on:
+  push:
+    branches:
+      - main
```

```suggestion
      - main
    paths:
      - 'src/reuse/**.py'
```

Besides the diff check after the fact, we can also ensure the Action is only run when relevant files have been edited.
GitHub Actions Documentation: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

The current list of files considered for translation is here: https://github.com/fsfe/reuse-tool/blob/main/po/POTFILES.in

---

done